docs/architecture.md

```diff
@@ -0,0 +1,43 @@
+# Bootleg Architecture Documentation
+
+Disclaimer: The purpose of the work here is to inspire others to try and implement something like what we've described. Building a full working system of this level of complexity would require a non-trivial amount of work. We believe the information here represents our best-guess at what we might build should we have the time/resources to do so. Our hope is that you will find parts of this project inspiring or thought provoking enough to dig in and experiment on your own. We are open to any questions, comments, feedback but keep in mind that this is a work of fiction.
+
+## System components
+
+We are imaging that the system to support the story defined in Bootleg would be composed of multiple systems. The systems are as follows:
+
+### Mobile Apps
+
+Fan App - The fan app will allow people to access and view the bootlegs created by the Bootleggers in the system. It will function similarly to a music store but instead of buying to own the bootlegs you are buying access and future royalties on sales. Through the fan app you would be able to search for artists you like, see the latest popular Bootlegs, gain access and watch audio/video of the concert. There is also a social component where you can connect with other fans, bootleggers, and artists on the platform.
+
+Bootlegger App - Used during the concert, the app would allow the person creating the Bootleg to caputer the audio/video recording of the concert. It would support all the functions around reqesting access to the band to become a bootlegger for them, handling any royalty agreements, promoting your existing Bootlegs through the platform, and connecting with fellow Bootleggers at/around the concert you are attending. Note: This app is also built to support desktop use so audio/video files can be edited using traditional DAW/Non-linear editing software.

```

Comment:
> Used during the concert, the app would allow the person creating the Bootleg to caputer the audio/video recording of the concert.

Even with the disclaimer, I think this might be a distraction. In my brain I'd assumed that the bootlegger would use their own equipment and it probably wouldn't be a phone. I was thinking it would be "non professional photography equipment" which is usually defined as (from my experience at shows) cameras without one of those big lenses (DSRLs basically)